components/apimgt/org.wso2.carbon.apimgt.rest.api.webserver/src/gen/java/org/wso2/carbon/apimgt/rest/api/webserver/ApiOriginFilter.java

```diff
@@ -0,0 +1,21 @@
+package org.wso2.carbon.apimgt.rest.api.webserver;
+
+import java.io.IOException;
+
+import javax.servlet.*;
```

Comment:
Add all the importing packages/classes without using * for imports.

Reply:
Fixed with https://github.com/RenukaFernando/carbon-apimgt/commit/88634c560fb8dfbf87211a105d87445605d8592f